lib/presto-client/index.js

```diff
@@ -1,15 +1,11 @@
 const { URL } = require('url') ;
+const http = require('follow-redirects/http');
+const https = require('follow-redirects/https');
 
-var adapterFor = (function() {
-    var adapters = {
-        'http:': require('follow-redirects/http'),
-        'https:': require('follow-redirects/https'),
-    };
-
-    return function(protocol) {
-        return adapters[protocol];
-    };
-}());
+var adapters = {
+    'http:': http,
+    'https:': https,
+};
```

Comment:
I needed the the "raw" `http` and `https` modules to do the agent assignment below, and so then doing:
```
const http = require('follow-redirects/http');
const https = require('follow-redirects/https');

var adapterFor = function(protocol) {
  return adapters[protocol];
}
```

and it felt like it'd be easier to just not have the `adapterFor` function at all and just index the `adapters` object directly. Let me know if you'd like me to bring back the helper function.